.github/actions/e2e/setup-cluster/action.yaml

```diff
@@ -269,3 +273,13 @@ runs:
       k8s_version: ${{ inputs.k8s_version }}
       git_ref: ${{ inputs.git_ref }}
       private_cluster: ${{ inputs.private_cluster }}
+  - name: log retention policy
+    shell: bash
+    if: ${{ inputs.private_cluster == 'false' }}
+    env:
+      CLUSTER_NAME: ${{ inputs.cluster_name }}
+    run: |
+      aws logs put-retention-policy --log-group-name /aws/containerinsights/$CLUSTER_NAME/application --retention-in-days 30
```

Comment:
What's the default retention policy that CloudWatch assigns to us? Is there a way to configure the retention directly through the addon config with eksctl?

Reply:
There is no retention policy set on the log stream and there is currently no way to configure the retention using eksctl.